src/main/resources/templates/subscription.html

```diff
@@ -262,13 +262,21 @@ <h3 class="modal-title text-center" id="formHeader">Subscription Form</h3>
                         <!--  AUTHENTICATION  -->
                         <div class="pt-3 form-group">
                             <label class="h5 pl-1 control-label font-weight-bold">Authorization</label>
+                            <img class="cursor-pointer" id="authInfo" alt="Auth Info"
+                                src="assets/images/information.png" data-toggle="tooltip" data-placement="top"
+                                title="This selector enables you to select differen authentication methods. <br>
+                                    NO_AUTH: No Authorization used<br>
```

Comment:
"No authentication" is maybe more suitable and looks better, my opinion.

Reply:
Done.